x-pack/plugins/beats_management/public/config_schemas.ts

```diff
@@ -310,6 +310,86 @@ const metricbeatModuleConfig: YamlConfigSchema[] = [
   },
 ];
 
+const heartbeatConfig: YamlConfigSchema[] = [
+  {
+    id: 'type',
+    ui: {
+      label: 'Type',
+      type: 'select',
+    },
+    options: [
+      {
+        value: 'http',
+        text: 'http',
+      },
+      {
+        value: 'icmp',
+        text: 'icmp',
+      },
+      {
+        value: 'tcp',
+        text: 'tcp',
+      },
+    ],
+    error: 'Please select a monitor type',
+    required: true,
+  },
+  {
+    id: 'name',
+    ui: {
+      label: 'Monitor Name',
+      type: 'input',
+    },
+    error: 'Please specify a valid monitor name',
```

Comment:
Please use a translation token for this. 
Also perhaps it would be good to define what valid is?